FormalConjectures/ErdosProblems/97.lean

```diff
@@ -74,7 +74,7 @@ Does every convex polygon have a vertex with no other 4 vertices equidistant fro
 -/
 @[category research open, AMS 52]
 theorem erdos_97 :
-    (∀ A : Finset ℝ², ConvexIndep A → ¬HasNEquidistantProperty 4 A) ↔ answer(sorry) := by
+    (∀ A : Finset ℝ², A.Nonempty ∧ ConvexIndep A → ¬HasNEquidistantProperty 4 A) ↔ answer(sorry) := by
```

Comment:
I think here its better to use implication as this seems to be the standard convention elsewhere in this repo.